src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days > 2 && days < 7) {
```

Comment:
Using `days > 2` means that the discount is not applied when `days` is exactly 3. According to the task description, the discount should be applied for 3 or more days. The condition should be `days >= 3`.